### PR TITLE
adding bwb API lookup to config

### PR DESCRIPTION
**Pull request by mekarpeles**
Adds bwb price checking to localhost

### CODE REVIEW

File: conf/openlibrary.yml [hunk 1]
```diff
@@ -162,6 +162,9 @@ ia_ol_xauth_s3:
     s3_key: XXX
     s3_secret: XXX
 
+bwb_api:
+    url: http://products.betterworldbooks.com/service.aspx?ItemId=
```
Comment (hornc) [inline conf/openlibrary.yml]: I was going to check whether this should be https -- looks like the endpoint only responds via http though.

**Comment by mekarpeles** [comment]
This will block @tabshaikh's testing (if we can merge it in later today, that would be great)

**Comment by cdrini** [comment]
@mekarpeles How is this used? I don't see the string `bwb_api` anywhere in the repo.

**Comment by mekarpeles** [comment]
bwb_api is used in core/vendors.py to get price information form BWB

**Comment by cdrini** [comment]
How is it used in core/vendors?!? core/vendors uses a module constant:

https://github.com/internetarchive/openlibrary/blob/master/openlibrary/core/vendors.py#L14

**Comment by mekarpeles** [comment]
You're right, this PR should be closed, sorry

**Pull request closed without merging**